cmu_graphics/shape_logic.py

```diff
@@ -1105,21 +1114,47 @@ def hitsShape(self, *arguments):
         myShapes = utils.getChildShapes(self)
         targetShapes = utils.getChildShapes(targetShape)
 
+        myShapesEdges = [shape.getEdges() for shape in myShapes]
+        targetShapesEdges = [shape.getEdges() for shape in targetShapes]
```

Comment:
Is getEdges slow enough that it would be worth skipping it if the bounds of the relevant shapes don't intersect?

Reply:
Looks like no!